bin/mustache_cli.ml

```diff
@@ -12,18 +12,23 @@ let load_file f =
   close_in ic;
   (Bytes.to_string s)
 
-let run json_filename template_filename =
+let run ?(prepend_json=ref false) json_filename template_filename =
```

Comment:
Conceptually it is not very nice to take a `bool ref` here (there is no reason why this function should be allowed to mutate its argument). It would be nicer to use just a boolean, and call it with `run ~prepend_json:!prepend_json` to access the reference value in the callsite.

I don't think this needs to be an optional parameter, given that it is passed explicitly in its single callsite. It is very useful to make it a named parameter, though (unnamed boolean parameters are often unclear at the callsite). So maybe just `~prepend_json`, a non-optional labeled argument?